cloudformation_templates/alarms/LoginFailurePercentAlarm.j2

```diff
@@ -0,0 +1,5 @@
+{% extends "alarms/base_alarm.j2" %}
+
+{% block description %}Too many 5xx{% endblock %}
```

Comment:
Yep - I think you have the messages round the wrong way - the Login Failure one should be here?